src/components/dialog/dialog.js

```diff
@@ -114,6 +115,16 @@ const VUIDialog = ({
         dialog.setAttribute('tabindex', 1);
         dialog.setAttribute('aria-hidden', false);
 
+        // Alerts don't have close buttons
+        if (hasCloseBtn && !isAlert) {
+            const closeBtnEl = createEl({
+                element: 'button',
+                className: 'js-dialog-btn-close',
+                type: 'button'
+            })
```

Comment:
How is the text node added and I guess we'd need a style class for the consumer to style?